proto/v1/initializer.proto

```diff
@@ -2,14 +2,12 @@ syntax = "proto3";
 
 package v1;
 
-option go_package = "./v1";
+import "v1/common.proto";
 
 service InitializerService {
   rpc Status(Empty) returns (StatusResponse);
 }
 
-message Empty {}
```

Comment:
It is best practice to always have a meaningful request and response body, therefor i would remove the Empty message and specify a StatusRequest message instead, same applies for ListBackups.

Reply:
Done.